mep/people/admin.py

```diff
@@ -436,8 +476,61 @@ class Media:
         ]
 
 
+
+
+class PersonResource(ModelResource):
+    def before_import(self, dataset, *args, **kwargs):
+        # lower and camel_case headers
+        dataset.headers = [x.lower().replace(' ','_') for x in dataset.headers]
+        # turn off indexing temporarily
+        IndexableSignalHandler.disconnect()
+        ViafSignalHandler.disconnect()
+
+    def after_import(self, *args, **kwargs):
+        super().after_import(*args, **kwargs)
+        # reconnect indexing signal handler
+        IndexableSignalHandler.connect()
+        ViafSignalHandler.connect()
+
+    def before_import_row(self, row, **kwargs):
+        # gender to one char
+        def fmt_gender(x):
+            x = str(x).strip()
+            return x[0].upper() if x else ''
+        row['gender']=fmt_gender(row.get('gender'))
+
+        def fmt_nation(x):
+            x=str(x).strip()
+            if not x or x=='[no country]': return
+            return x
+        row['nation'] = fmt_nation(row.get('nation'))
+
+    def get_import_fields(self):
+        return [self.fields[fname] for fname in self.fields if fname in PERSON_IMPORT_COLUMNS]
+
+    name = Field(column_name='name', attribute='name')
+    gender = Field(column_name='gender', attribute='gender')
+    nationalities = Field(
+        column_name='nationalities',
+        attribute='nationalities',
+        widget=ManyToManyWidget(Country, field='name', separator=';')
+    )
+
+    class Meta:
+        model = Person
+        fields = PERSON_IMPORT_COLUMNS
+        import_id_fields = ('slug',)
+        export_order = PERSON_IMPORT_COLUMNS
```

Comment:
Write now export only exports the columns that are imported. We can change this (though there's annoyingly not a simple way to do this it appears in dj-imp-exp) or just document that for now. For instance if we wanted to annotate profession we could add that to both and write a little `profession = Field(...ForeignKey(..))` in the code above

Reply:
Got this working in latest, we export most everything we display in the Person table, and import only what we need